baselines/moon/README.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Model-Contrastive Federated Learning
+url: https://arxiv.org/abs/2103.16257
+labels: [data heterogeneity, image classification, cross-silo, constrastive-learning]
+dataset: [CIFAR-10, CIFAR-100]
+---
+
+# Model-Contrastive Federated Learning
+
+> Note: If you use this baseline in your work, please remember to cite the original authors of the paper as well as the Flower paper.
+
+
+****Paper:**** : [arxiv.org/abs/2103.16257](https://arxiv.org/abs/2103.16257)
```

Comment:
There is a repeated `:` after the **bold** text. This issue happens other times below.